OpenEdXMobile/src/main/java/org/edx/mobile/view/MainTabsDashboardFragment.java

```diff
@@ -59,7 +59,9 @@ public void onCreate(Bundle savedInstanceState) {
         final boolean isUserProfileEnabled = environment.getConfig().isUserProfilesEnabled();
         if (isUserProfileEnabled) {
             profile = loginPrefs.getCurrentUserProfile();
-            sendGetUpdatedAccountCall();
+            if (profile != null) {
```

Comment:
This seemed like it should be updated, but I made the change out of context, @miankhalid @farhan Does this look like the right check here?

Reply:
As long as this change comes in a separate commit which isn't squashed at the end will all the other changes, it should be fine.